### DISCUSSION TITLE
想问下，这个工程现在YOLOV8有没有详细的使用教程

### DISCUSSION THREAD

**Discussion by hukaick** [positive666/yolov5_research#94] [Ideas]
您好，想问下，这个工程现在YOLOV8有没有详细的使用教程，怎么修改自己模型定义和训练

**Comment by positive666** [comment]
> 您好，想问下，这个工程现在YOLOV8有没有详细的使用教程，怎么修改自己模型定义和训练

因为V8的代码风格是站在使用者的”低代码“的层面去设计的，所以如果自己改的话取决你改的部分，这个项目的核心是以YOLOV5的架构为主，V8单独放在一个yolo的文件夹下，这样一是好改一些且作者更新太快，我可以把问题只锁定在那一部分代码周边可以快速debug排查，也就是说目前V8的训练部分主要是检测功能的程序是单独和V5分离训练的，但是模型的结构定义和推理都是兼容的，有可能会比较麻烦，但这只是时间问题，早期的CSDN上介绍过了，目前有人反馈train_v8.py 不能加载自定义模型的权重，抽时间会补更的，最近事情比较多

**Reply by hukaick** [reply]
我以前按照您的V5修改过BIFPN，在实验中我的数据集提了约2个点，但是后来发现有些参数没有更新，随着您的代码更新发现您改动非常大，有点不好理解
